saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -1636,6 +1636,52 @@ def test_checkout_no_available_shipping_methods_without_lines(api_client, checko
 }
 """
 
+QUERY_GET_ALL_COLLECTION_POINTS_FROM_CHECKOUT = """
+        query AvailableCollectionPoints(
+            $token: UUID!
+            ){
+            checkout(token: $token){
+    availableCollectionPoints{
+      name
+    }
+  }
+}
+    """
+
+
+def test_available_collection_points_for_preorders_variants(
+    api_client, staff_api_client, checkout_with_preorders_only
+):
+
+    expected_collection_points = list(
+        Warehouse.objects.for_country("US").values("name")
+    )
+    response = staff_api_client.post_graphql(
+        QUERY_GET_ALL_COLLECTION_POINTS_FROM_CHECKOUT,
+        variables={"token": checkout_with_preorders_only.token},
+    )
+    decoded_response = response.json()
```

Comment:
I don't think response from the server is encoded. I suggest to change this variable name.

Reply:
We use `get_graphql_content` for that.